<ui version="4.0" >
 <class>GuiKeyboardSetupDialog</class>
 <widget class="QDialog" name="GuiKeyboardSetupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>348</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>501</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Setup Your Piano Keyboard</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QTextEdit" name="keyboardInfoText" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="title" >
           <string>Right Notes</string>
          </property>
          <layout class="QHBoxLayout" >
           <item>
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label_4" >
               <property name="text" >
                <string>sound:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QComboBox" name="rightSoundCombo" >
               <property name="minimumSize" >
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QLabel" name="rightVolumeLabel" >
               <property name="text" >
                <string>volume:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <layout class="QHBoxLayout" >
               <item>
                <widget class="QSpinBox" name="rightVolumeSpin" >
                 <property name="suffix" >
                  <string>%</string>
                 </property>
                 <property name="minimum" >
                  <number>1</number>
                 </property>
                 <property name="maximum" >
                  <number>200</number>
                 </property>
                 <property name="value" >
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="rightTestButton" >
                 <property name="text" >
                  <string>Test</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="title" >
           <string>Wrong Notes</string>
          </property>
          <layout class="QHBoxLayout" >
           <item>
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label" >
               <property name="text" >
                <string>sound:</string>
               </property>
              </widget>
             </item>
             <item rowspan="2" row="0" column="1" >
              <widget class="QComboBox" name="wrongSoundCombo" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item rowspan="2" row="1" column="0" >
              <widget class="QLabel" name="wrongVolumeLabel" >
               <property name="text" >
                <string>volume:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" >
              <layout class="QHBoxLayout" >
               <item>
                <widget class="QSpinBox" name="wrongVolumeSpin" >
                 <property name="suffix" >
                  <string>%</string>
                 </property>
                 <property name="minimum" >
                  <number>1</number>
                 </property>
                 <property name="maximum" >
                  <number>200</number>
                 </property>
                 <property name="value" >
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="wrongTestButton" >
                 <property name="text" >
                  <string>Test</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="title" >
         <string>Keyboard Note Range</string>
        </property>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Lowest Note:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lowestNoteEdit" >
           <property name="maximumSize" >
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>The note number between 0 and 127</string>
           </property>
           <property name="inputMask" >
            <string>000; </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Highest Note:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="highestNoteEdit" >
           <property name="maximumSize" >
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>The note number between 0 and 127</string>
           </property>
           <property name="inputMask" >
            <string>000; </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton" >
           <property name="text" >
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GuiKeyboardSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GuiKeyboardSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
